web/admin/src/SessionBox.js

```diff
@@ -99,10 +100,21 @@ export class SessionBox extends Component {
 
     handleNewSession = () => {
       const current = this.state.showNewSession
-      this.setState({showNewSession: !current})
+      this.setState({showNewSession: !current, color: "#FAFAFA"})
     }
 
     updateList = (value) => {
+      var queryText = value.toLowerCase()
+      if (queryText.length > 0){
```

Comment:
Since boolean are values., this whole if/else (6 lines) can be replaced with `this.setState({search: queryText.length > 0, searchValue: value})`